projects/core/components/textfield/textfield-auxiliary.ts

```diff
@@ -11,7 +11,7 @@ export function tuiAsAuxiliary(x: Type<unknown>): Provider {
 }
 
 export function tuiInjectAuxiliary<T>(
-    predicate: (auxiliary: Type<unknown>, index: number) => boolean,
+    predicate: (auxiliary: any, index: number) => boolean,
```

Comment:
Actually, already created instance of component (queried by `@ContentChild`) is no more `Type<T>`-compatible